README.md

```diff
@@ -16,7 +16,7 @@
 
 <p align="center">
   <a href="https://scrumlr.io">
-    <img src="./public/screenshots/board.png" alt="scrumlr.io" />
+    <img src="board.png" alt="scrumlr.io" />
```

Comment:
do we require the `./public/screenshots/` folder for anything or is it obsolete now?

Reply:
for what? I can't find a use in the project 😅

Reply:
alright, I'm just gonna believe that then 🤷‍♂️😄